py/src/call.py

```diff
@@ -142,6 +153,30 @@ def loop_inner(connection, command):
         )
     )
 
+    if verb == "call":
+        return (verb, result.call_info.retdata)
+    else:
+        assert verb == "estimate_fee", "command should had been call or estimate_fee"
```

Comment:
Can't we have this check even before processing the command? We could have verb parsed into an `Enum` for example right at the beginning and then use the enum type:

```
In [3]: from enum import Enum, auto

In [4]: class Verb(Enum):
   ...:     call = auto()
   ...:     estimate = auto()
   ...: 

In [5]: Verb["call"]
Out[5]: <Verb.call: 1>

In [6]: Verb["auto"]
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
Input In [6], in <cell line: 1>()
----> 1 Verb["auto"]

File /usr/lib/python3.10/enum.py:440, in EnumMeta.__getitem__(cls, name)
    439 def __getitem__(cls, name):
--> 440     return cls._member_map_[name]

KeyError: 'auto'
```

Reply:
The command "parser" verifies the input is part of the set. I don't think we should try to add much more typing to python with more constructs.